float64 t

float64 des_x
float64 des_y
float64 des_z

float64 ref_x
float64 ref_y
float64 ref_z

float64 est_x
float64 est_y
float64 est_z

float64 des_vx
float64 des_vy
float64 des_vz

float64 ref_vx
float64 ref_vy
float64 ref_vz

float64 est_vx
float64 est_vy
float64 est_vz
